babi/buf.py

```diff
@@ -299,6 +299,32 @@ def left(self, margin: Margin) -> None:
         else:
             self.x -= 1
 
+    def paragraph_down(self, margin: Margin) -> None:
```

Comment:
I played with this a little bit, and the behaviour is a little different from how the ^Arrow keys ones work

I think it would make sense to have two stop points, one at the end of the paragraph and one at the beginning of the next paragraph rather than the current behaviour which is just the beginning of paragraph